contracts/Controller.sol

```diff
@@ -484,4 +473,13 @@ contract Controller is IController {
         feeToken.transfer(lockFees, _balance);
         IRewards(lockFees).queueNewRewards(_balance);
     }
+
+    /// @notice redirects rewards from gauge to rewards contract
+    /// @param _pid the id of the pool
+    function redirectGaugeRewards(uint256 _pid) private {
```

Comment:
Is it possible that we change the stash? If it is, maybe make it public?

Reply:
it is created in `addPool` via factory, and can't be changed